ocean_provider/run.py

```diff
@@ -25,11 +25,31 @@ def get_version():
 
 @app.route("/")
 def version():
+    """
+    Contains the data necessary for user provider:
+        - software;
+        - version;
+        - network url;
+        - provider address;
+        - a dictionary with the existing endpoints
+        from routes.py;
+    """
+
     info = dict()
     info['software'] = Metadata.TITLE
     info['version'] = get_version()
     info['network-url'] = config.network_url
     info['provider-address'] = get_provider_wallet().address
+    # Added the expose endpoint. Here are the URLs to the
+    # existed endpoints. Check test_compute.py, in function
+    # test_compute_expose_endpoint.
+    info['servicesEndpoints'] = {
+        "access": url_for('services.download'),
+        "compute": url_for('services.compute_get_status_job'),
+        "nonce": url_for('services.get_user_nonce'),
+        "encrypt": url_for('services.encrypt'),
+        "initialize": url_for('services.initialize'),
+    }
```

Comment:
does this include all the endpoints?

Reply:
missing: 
compute_start_job, compute_stop_job and compute_delete_job

Reply:
Ok, thanks. I will modify that issue too.